examples/javascript/index.js

```diff
@@ -0,0 +1,140 @@
+var math = require('mathjs');
+
+data1 = [
+  [0, 10, 10, 80, 0],
+  [10, 0, 10, 0, 80],
+  [10, 10, 0, 40, 40],
+  [0, 0, 0, 1, 0],
+  [0, 0, 0, 0, 1],
+];
+
+data2 = [
+    [  1,   0,   0,   0,   0],
+    [0.5,   0, 0.5,   0,   0],
+    [  0, 0.5,   0, 0.5,   0],
+    [  0,   0, 0.5,   0, 0.5],
+    [  0,   0,   0,   0,   1],
+];
+
+function range(size, start_at= 0) {
+    return [...Array(size).keys()].map(i => i + start_at);
```

Comment:
lol nice. does math.js not have a range function?

Reply:
Yeah, but it returns a `Matrix`, which doesn't work with where it's used :P